tournament/on.cfg

```diff
@@ -1,6 +1,7 @@
+sv_vote_checkboxes 1
```

Comment:
This should be 0 to make sure loading the config does not cause lags. Its a bit of a hack.

I can also fix it don't worry. Are you done? Then I would merge now.

Reply:
Yea i hope so, thanks :D